testrail_api/_session.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         url: Optional[str] = None,
         email: Optional[str] = None,
         password: Optional[str] = None,
+        session: Optional[requests.Session] = None
```

Comment:
Maybe a comma is needed here?

Reply:
it would be nice for users if the kwarg were added at the end:

        retry_exceptions: Tuple[Type[BaseException], ...] = (),
        response_handler: Callable[[requests.Response], Any] = None,
        session: Optional[requests.Session] = None,
        **kwargs,